### PR TITLE
fixed copyright template for idea

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
copyright template idea

### Description
This PR fixes the template to always use the current year.

### Steps

- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  signed-off my commits to accept the DCO.

### CODE REVIEW

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]
According to https://stackoverflow.com/questions/30618147/how-do-you-insert-the-current-year-into-intellij-idea-copyright-template/30618447
it should be

`${today.year}`

**Comment by DerMolly** [comment]
I tested it and this works, but if you want I can change this.

**Comment by mrdrogdrog** [comment]
Oh it does? Okay, good, then you can see it as suggestion. But tbh I would still prefer to capsule it, so it's clearer to recognize it as template stuff.

(Funfact: It's also good practice in bash to prevent misinterpretation by the runtime)

**Review by mrdrogdrog — APPROVED** [review]

File: .idea/copyright/hedgedoc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <component name="CopyrightManager">
   <copyright>
-    <option name="notice" value="SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)&#10;&#10;&#83;PDX-License-Identifier: AGPL-3.0-only" />
+    <option name="notice" value="SPDX-FileCopyrightText: $today.year The HedgeDoc developers (see AUTHORS file)&#10;&#10;&#83;PDX-License-Identifier: AGPL-3.0-only" />
```
Comment (ErikMichelson) [inline .idea/copyright/hedgedoc.xml:3]: I agree with @mrdrogdrog, that the brackets would look more clear here.

Comment (DerMolly) [reply .idea/copyright/hedgedoc.xml]: done

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**